### PR TITLE
Fixes #13284 - Enables Taskgroups for Applicablity

### PR DESCRIPTION
**Pull request by parthaa**
The regenerate applicable errata  call for a repository can return
either a Task Group or a list of spawned tasks. This commit enables the
use of that. This is an important change in pulp 2.8. Only available
there.

### CODE REVIEW

**Comment by parthaa** [comment]
Broke it in to 2 commits. One containing the code. Other the fixture.

**Comment by parthaa** [comment]
This is an important change in pulp 2.8.
Look at
https://github.com/pulp/pulp/blob/master/docs/dev-guide/integration/rest-api/consumer/applicability.rst#generate-content-applicability-for-updated-repositories
for more info.

File: lib/runcible/resources/task_group.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+module Runcible
```
Comment (ehelms) [inline lib/runcible/resources/task_group.rb]: Tests missing for this file

File: lib/runcible/resources/task_group.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      # summary for the status of a task
+      #
+      # @param  [String]              id  the id of the task
+      # @return [RestClient::Response]
+      def summary(id)
+        call(:get, self.class.summary_path(id))
+      end
+
+      def completed?(id)
+        report = summary(id)
```
Comment (ehelms) [inline lib/runcible/resources/task_group.rb]: I know we discussed whether to split this up into two functions but couldn't remember where it landed.

Comment (parthaa) [inline lib/runcible/resources/task_group.rb]: Oh right!. Forgot all about that. Thanks for reminding.

**Comment by parthaa** [comment]
@ehelms updated

File: lib/runcible/resources/task_group.rb (hunk 1 above)
Comment (parthaa) [inline lib/runcible/resources/task_group.rb]: https://github.com/parthaa/runcible/commit/ddfc4136295a28652a1aeb3b058689662221db5d#diff-b53809ac5865ab571f5b842e06e123b2

**Comment by ehelms** [comment]
This is linked to a bug in Katello and not a feature in Runcible -- can you please open an issue for this and update the commits?

**Comment by parthaa** [comment]
@ehelms I made a separate bug for Katello and changed project for this to runcible.

**Comment by parthaa** [comment]
@ehelms  katello issue now being tracked at http://projects.theforeman.org/issues/13566

**Comment by ehelms** [comment]
ACK - Thanks @parthaa !

**Pull request merged**